docs/release-notes/release-notes-0.14.2.md

```diff
@@ -6,10 +6,15 @@
   install](https://github.com/lightningnetwork/lnd/pull/6035).
 
 * [Make etcd max message size
-  configurable]((https://github.com/lightningnetwork/lnd/pull/6049).
+  configurable](https://github.com/lightningnetwork/lnd/pull/6049).
+
+* [Export bitcoind port and other values for itests, useful for
+  using itest harness outside of lnd]
+  (https://github.com/lightningnetwork/lnd/pull/6050).
```

Comment:
nit: there shouldn't be a newline between `]` and `(` for markdown links, otherwise they don't render. So best to break before the last word (`lnd](https://github.com...`).